prover/crates/bin/prover_autoscaler/src/k8s/scaler.rs

```diff
@@ -0,0 +1,27 @@
+use k8s_openapi::api;
+use kube::api::{Api, Patch, PatchParams};
+
+#[derive(Clone)]
+pub struct Scaler {
+    pub client: kube::Client,
+}
+
+impl Scaler {
+    pub async fn scale(&self, namespace: &str, name: &str, size: i32) -> anyhow::Result<()> {
+        let deployments: Api<api::apps::v1::Deployment> =
+            Api::namespaced(self.client.clone(), namespace);
+
+        let patch = serde_json::json!({
+            "apiVersion": "apps/v1",
+            "kind": "Deployment",
+            "spec": {
+                "replicas": size
+            }
+        });
+        let pp = PatchParams::default();
+        let _patched = deployments.patch(name, &pp, &Patch::Merge(patch)).await?;
```

Comment:
You can do `let _ = deployments.patch(name, &pp, &Patch::Merge(patch)).await?;` if the value is not needed, or even avoid let instruction at all

Reply:
Removed.